app/javascript/app/providers/login-provider/login-provider-actions.js

```diff
@@ -17,3 +25,15 @@ export const getUser = createThunkAction('getUser', () => (dispatch, state) => {
       });
   }
 });
+
+export const saveUserData = createThunkAction(
+  'saveUserData',
+  () => (dispatch, getState) => {
+    const { login } = getState();
+    CWAPI.patch(`my_cw/users/${login.user.user_id.id}`, login.user.user_id)
+      .then(dispatch(profileUpdated()))
+      .catch(e => {
+        console.warn(e);
```

Comment:
It would be nice to show some error too when the profile fails to update. I think its even more valuable than the confirm that everything worked